content/en/docs/contribute/review/reviewing.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Reviewing pull requests
+content_template: templates/concept
+main_menu: true
+weight: 10
+---
+
+{{% capture overview %}}
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/kubernetes/website/pulls) section in the Kubernetes website repository to see open pull requests.
+
+Reviewing documentation pull requests is a
```

Comment:
Line 12: Yes, I would argue that reviewing is better than opening an issue -- reading and writing code+docs.